ajax/libs/youtube-google-analytics/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "youtube-google-analytics",
+  "filename": "lunametrics-youtube.gtm.min.js",
+  "version": "8.1.0",
+  "description": "This is a plug-and-play tracking solution for tracking user interaction with YouTube videos in Google Analytics. It will detect if GTM, Universal Analytics, or Classic Analytics is installed on the page, in that order, and use the first syntax it matches unless configured otherwise. It include support for delivering hits directly to Universal or Classic Google Analytics, or for pushing Data Layer events to be used by Google Tag Manager.",
+  "homepage": "https://github.com/lunametrics/youtube-google-analytics",
```

Comment:
@KZeni We'll use git repo as homepage by default if the custom homepage is not found, so let's remove homepage field.